src/startup.rs

```diff
@@ -95,3 +81,22 @@ fn status_list_routes(state: AppState) -> Router<AppState> {
         .merge(protected_routes)
         .route("/{list_id}", get(get_status_list))
 }
+
+fn metrics(router: Router, config: &Config) -> Router {
+    if config.server.enable_metrics {
+        match crate::utils::metrics::setup_metrics() {
```

Comment:
Import `crate::utils::metrics` so the dependencies are clear.
```suggestion
        match metrics::setup_metrics() {
```